accounts/tests.py

```diff
@@ -1,3 +1,43 @@
 from django.test import TestCase
+from accounts.forms import UserRegisterForm
+from django.contrib.auth import get_user_model
+User = get_user_model()
 
-# Create your tests here.
+
+class SignUpTests(TestCase):
+
+    def test_signup_view(self):
+        response = self.client.post('/accounts/signup/')
+        self.assertEqual(response.status_code, 200)
```

Comment:
should this be a 200 OK?  If I POST no data, I would expect a `400` back - no data is an invalid request!